### ISSUE TITLE
Can't install latest version

### ISSUE THREAD

**Issue by erwanlr** [wpscanteam/homebrew-tap#2]
Tried on macOSX Catalina

```
An error occurred while installing nokogiri (1.10.10), and Bundler cannot continue.
Make sure that `gem install nokogiri -v '1.10.10' --source 'https://rubygems.org/'` succeeds before bundling.

In Gemfile:
  wpscan was resolved to 3.8.11, which depends on
    cms_scanner was resolved to 0.12.1, which depends on
      nokogiri
```

I think it's due to ruby 3.0.0 being installed and Nokogiri not supporting that one yet.

I tried to force the ruby version required to 2.7.2 w/o success :/

Also, can't see the details of the error, because the log file that brew recommends to check, ie `/usr/local/Cellar/wpscan/3.8.11/libexec/extensions/x86_64-darwin-19/3.0.0/nokogiri-1.10.10/mkmf.log` does not exist

I have no issue with installing `Nokogiri` via `gem install nokogiri` manually in my system though (with ruby being in RVM):
```
% gem install nokogiri
Building native extensions. This could take a while...
Successfully installed nokogiri-1.10.10
1 gem installed
```

**Comment by erwanlr** [comment]
@ran-dall Any idea about this one please ? I've tried numerous things w/o success :/

**Comment by erwanlr** [comment]
Has been fixed with the release of Nokogiri 1.11.0
